R/get_r_messages.R

```diff
@@ -274,10 +281,24 @@ exclude_untranslated = function(expr_data, comments) {
 #     if (is.null(f_args <- args(f))) next
 #     if (any(names(formals(f_args)) == 'domain')) cat(obj, '\n')
 # }
-DOMAIN_DOTS_FUNS = c("warning", "stop", "message", "packageStartupMessage", "gettext")
+domain_dots_funs <- function(include_conditions = TRUE) {
+  c(
+    "gettext",
+    if (include_conditions) c("stop", "warning", "message", "packageStartupMessage")
+  )
+}
+
+domain_fmt_funs <- function(include_conditions = TRUE) {
+  c(
+    "gettextf",
+    if (include_conditions) c("stopf", "warningf", "messagef")
```

Comment:
`packageStartupMessagef` too right? We added that in `data.table` which can have had some gnarly messages before templating...

https://github.com/Rdatatable/data.table/blob/master/R/translation.R

https://github.com/Rdatatable/data.table/blob/e88826e94127991754a8821ac8149d15f1f9c3c2/R/onAttach.R#L27-L36

Reply:
I'll make this build on `domain_dots_funs()` to avoid any chance on inconsistency